tests/cases/conformance/es2022/es2024SharedMemory.ts

```diff
@@ -0,0 +1,26 @@
+// @target: esnext
+// @lib: es2022
+// @noemit: true
+// @strict: true
+
+// ES2024 Atomics.wait was included in the ES2022 type file due to a mistake.
```

Comment:
```suggestion
// ES2024 Atomics.waitAsync was included in the ES2022 type file due to a mistake.
```

Reply:
My apologies, this will also require re-running the tests to capture test outputs that include the comment.